baybe/kernels/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""Kernels that can be used for GP surrogate models."""
```

Comment:
Would shorten + not have abbreviations in there
```suggestion
"""Kernels for Gaussian process surrogate models."""
```